socs/agents/acu/agent.py

```diff
@@ -2087,8 +2087,7 @@ def _notify_recomputed(result):
     @ocs_agent.param('avoidance_radius', type=float, default=None)
     @ocs_agent.param('shift_sun_hours', type=float, default=None)
     def update_sun(self, session, params):
-        """update_sun(reset, enable, temporary_disable, escape,
-                      avoidance_radius, shift_sun_hours)
+        """update_sun(reset, enable, temporary_disable, escape, avoidance_radius, shift_sun_hours)
```

Comment:
Line wraps are possible here with a `\` if needed. (I noticed the wrapped line to unwrapped line change in the few commits.) (See the edit on the stream docstring in [this PR](https://github.com/simonsobs/socs/pull/553/files) for example.)